cotton/salt/tasks.py

```diff
@@ -54,17 +57,16 @@ def highstate_complete():
     Poll jobs active waiting for it to become empty after an unattended_highstate was started
     """
     timeout = 15
-    from cStringIO import StringIO
     result = StringIO()
     salt_run(method='jobs.active', stdout=result)
 
     while len(result.getvalue().strip()):
         result.truncate(0)
-        print(yellow("Highstate is still running.\nPolling again in {} seconds.\n".format(timeout)))
+        print yellow("Highstate is still running.\nPolling again in {} seconds.\n".format(timeout))
         time.sleep(timeout)
         salt_run(method='jobs.active', stdout=result)
 
-    print(green("Highstate complete.\n"))
+    print green("Highstate complete.\n")
 
```

Comment:
yeap, you can disable those  print-statement (E1601)
but not really worth spending more time on this